src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -160,6 +169,13 @@ export const SueReportLocation = ({
     [data]
   );
 
+  // TODO: do we need this?
+  // useFocusEffect(() => {
+  //   if (selectedPosition) {
+  //     setSelectedPosition(selectedPosition);
+  //   }
+  // });
```

Comment:
don't miss this, so I add a comment here

Reply:
No. We don't need it anymore because we don't switch to another screen for the big map view. We do everything from one map.